test/jit/test_complex.py

```diff
@@ -14,6 +14,13 @@ def fn(a: complex):
 
         self.checkScript(fn, (3 + 5j,))
 
+    def test_complexlist(self):
+        def fn(a: List[complex], idx: int):
+            return a[idx]
+
+        input = [1j, 2, 3 + 4j, -5, -7j]
```

Comment:
What about serialization of lists? Is that coming later?

Reply:
will add it in a follow-up PR